playbooks/prosody.yml

```diff
@@ -1,12 +1,17 @@
-- hosts: solr_prod
+---
+# by default this playbook runs in the qa environment
+# to run in production, pass '-e runtime_env=prod'
```

Comment:
same thing here - this means it will only run correctly for staging/qa if you rely on the default - you can't pass `runtime_env=qa` because we don't have a qa env for solr